packages/core/src/shared/telemetry/spans.ts

```diff
@@ -97,6 +98,9 @@ export type SpanOptions = {
     /** True if this span should emit its telemetry events. Defaults to true if undefined. */
     emit?: boolean
 
+    /** True if this span should emit performance metrics acquired when running the function. Defaults to false if undefined */
+    trackPerformance?: boolean
```

Comment:
Is there a reason this can't be a field on the metric itself (or MetricBase or whatever)? I don't quite understand why we need SpanOptions. We treat `passive` like an "option", so why not have other fields that work like that?